pkg/catalog/retry_test.go

```diff
@@ -14,9 +15,14 @@ import (
 	"github.com/openservicemesh/osm/pkg/k8s"
 	"github.com/openservicemesh/osm/pkg/policy"
 	"github.com/openservicemesh/osm/pkg/service"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
It looks like linter is complaining about these not being sorted. You could move this to the section above since it is not an openservicemesh package.